distributed/dashboard/components/scheduler.py

```diff
@@ -1193,6 +1194,7 @@ def update(self):
 
     @without_property_validation
     def add_new_nodes_edges(self, new, new_edges, update=False):
+        max_items = config.get('distributed.dashboard.graph_max_items', 2)
```

Comment:
For config we typically do the following by convention

```suggestion
        max_items = dask.config.get('distributed.dashboard.graph_max_items', 2)
```